tests/sentry/api/test_id_or_slug_path_parameters.py

```diff
@@ -278,6 +322,61 @@ def organization_test(self, endpoint_class, slug_params, *args):
 
         self.assert_conversion(endpoint_class, converted_slugs, converted_ids)
 
+    @patch("sentry.api.bases.organization.OrganizationEndpoint.check_object_permissions")
+    @patch("sentry.types.region.subdomain_is_region")
+    @override_options({"api.id-or-slug-enabled": True})
```

Comment:
Oh I just realized we're always gonna want this true for all the tests in this file. You can use a pytest fixture do make a function that does that like this:
https://github.com/getsentry/sentry/blob/483456bfea318484569667dadf8820b5e351e578/tests/sentry/api/endpoints/test_user_details.py#L315-L318